backup/setup/helpdocuments.json

```diff
@@ -125,7 +125,7 @@
+				"content": "<a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Anlagen-zum-Vertrag.pdf'>Anlagen zum Vertrag [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Datenschutz_Checkliste.pdf'>Datenschutz-Checkliste [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Datenschutzerklaerung-Muster-Schulen.pdf'>Datenschutzerklärung Muster Schulen [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Dienstvereinbarung-Schule-zum-HPI.pdf'>Dienstvereinbarung [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Einwilligungserklaerung-Muster-Schulen.pdf'>Einwilligungserklärung Muster Schulen [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Einwilligungskonzept.pdf'>Einwilligungskonzept [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/Nutzungsordnung_Schueler-innen.pdf'>Nutzungsordnung zwischen Schule und Schüler [.pdf]</a><br>  <a target='_blank' rel='noopener' href='https://schul-cloud-hpi.s3.hidrive.strato.com/default/Willkommensordner/Datenschutz/VVT-Muster-Schulen.pdf'>VVT Muster Schulen [.pdf]</a>"
```

Comment:
why to keep schul-cloud-hpi in subdomain but change it in path?

Reply:
you are right, forgot to change it on this position as we do not change every single file at once, but I need to change some more subdomains, thx!